buildspecs/core.feature

```diff
@@ -119,6 +119,7 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-ex
 		<flag id="opt_openjdkFfi" value="true"/>
 		<flag id="opt_openjdkMethodhandle" value="false"/>
 		<flag id="opt_phpSupport" value="false"/>
+		<flag id="opt_portableSharedCache" value="true"/>
```

Comment:
Please don't do this (see https://github.com/eclipse-openj9/openj9/pull/19399#discussion_r1583339146).
In its current form, I don't think `J9VM_OPT_PORTABLE_SHARED_CACHE` will be defined anywhere (at least not in cmake builds).